src/components/strategies/StrategyPageTabs.tsx

```diff
@@ -15,16 +15,27 @@ interface Props {
 }
 
 export const StrategyPageTabs = ({ currentPathname, tabs }: Props) => {
+  const match = useMatchRoute();
   return (
     <nav
       aria-label="Strategy Panels"
       className="flex w-full gap-2 rounded-full border-2 border-silver p-6 text-14 md:w-auto"
     >
-      {tabs.map(({ label, href, icon, badge, hrefMatches }) => {
-        const active = isPathnameMatch(currentPathname, href, hrefMatches);
+      {tabs.map(({ label, href, params, icon, badge }) => {
+        const active = match({
+          to: href,
+          search: {},
+          params: params ?? {},
+          fuzzy:
+            currentPathname.includes('/token/') && href.includes('portfolio'),
+        });
+
         return (
           <Link
             to={href}
+            // TODO: fix this
+            params={params ?? {}}
+            search={{}}
```

Comment:
Same as above. Do we need an issue for that ?

Reply:
#998